docs/configuration.md

```diff
@@ -278,11 +278,14 @@ domain must be separated by a comma.
       mapping:
         printer.lan: 192.168.178.3
         otherdevice.lan: 192.168.178.15,2001:0db8:85a3:08d3:1319:8a2e:0370:7344
+        anothername.lan: CNAME(otherdevice.lan)
     ```
 
 This configuration will also resolve any subdomain of the defined domain, recursively. For example querying any of
 `printer.lan`, `my.printer.lan` or `i.love.my.printer.lan` will return 192.168.178.3.
 
+CNAME records are supported by setting the value of the mapping to `CNAME(target)`. Note that the target will be recursively resolved and there are no checks for CNAME loops, meaning that it is possible for CNAMES to never resolve due to a loop.
```

Comment:
No breaker but just an idea:
If every CNAME that is looped through is added to a slice you could check if it already was in the resolving CNAME chain and break the loop with an error. 🤔

Reply:
Great idea! I just pushed up a change to add this to the custom resolver!

Reply:
But it broke the linter :man_facepalming: I'll have that fixed in a few mintues